mk/generate.mk

```diff
@@ -29,10 +29,10 @@ clean/protos: ## Dev: Remove auto-generated Protobuf files
 generate: generate/protos $(if $(findstring ./api,$(PROTO_DIRS)),resources/type generate/builtin-crds) generate/policies $(EXTRA_GENERATE_DEPS_TARGETS) ## Dev: Run all code generation
 
 $(POLICY_GEN):
-	cd $(KUMA_DIR) && go build -o ./build/tools/policy-gen/generator ./tools/policy-gen/generator/main.go
+	cd $(KUMA_DIR) && go build -o $(POLICY_GEN) ./tools/policy-gen/generator/main.go
```

Comment:
is this correct that it's now always in `KUMA_DIR`?

Reply:
I'm a bit confused. Let's look at the invocation of this binary
```
.PHONY: resources/type
resources/type: $(RESOURCE_GEN)
	$(RESOURCE_GEN) -package mesh -generator type > pkg/core/resources/apis/mesh/zz_generated.resources.go
	$(RESOURCE_GEN) -package system -generator type > pkg/core/resources/apis/system/zz_generated.resources.go


generate/kumapolicy-gen/%: $(POLICY_GEN) generate/dirs/%
	$(POLICY_GEN) core-resource --plugin-dir $(POLICIES_DIR)/$* --gomodule $(GO_MODULE) && \
	$(POLICY_GEN) k8s-resource --plugin-dir $(POLICIES_DIR)/$* --gomodule $(GO_MODULE) && \
	$(POLICY_GEN) openapi --plugin-dir $(POLICIES_DIR)/$* --openapi-template-path=$(TOOLS_DIR)/policy-gen/templates/endpoints.yaml --gomodule $(GO_MODULE) && \
	$(POLICY_GEN) plugin-file --plugin-dir $(POLICIES_DIR)/$* --gomodule $(GO_MODULE) && \
	$(POLICY_GEN) helpers --plugin-dir $(POLICIES_DIR)/$* --gomodule $(GO_MODULE)
```

Wasn't it incorrect before?

Reply:
ah we run `cd`, so actually it needs be without `KUMA_DIR` as before. more correct would probably be to trim `KUMA_DIR` as prefix from `RESOURCE_GEN`/`POLICY_GEN`